packages/ui/src/helpers/formatBalance.ts

```diff
@@ -0,0 +1,24 @@
+export interface Balance {
+  amount: bigint
+  decimals: number
+  symbol: string
+}
+
+export function formatBalance(
```

Comment:
can we use the same approach here as in argentx?

Reply:
sure, updated @vladutjs